Jamfile

```diff
@@ -376,6 +377,12 @@ rule building ( properties * )
 		}
 	}
 
+	if [ configure.builds /boost/architecture//x86 : $(properties) : x86 ]
+		&& ! <target-os>windows in $(properties)
```

Comment:
I will put `! <toolset>msvc` instead, since this is an option for gcc/clang and who knows all the name combinations